great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -309,6 +310,15 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
 
         self._sources: _SourceFactories = _SourceFactories(self)
 
+        self._suites: Union[SuiteFactory, None]
+        if self.stores.get(self.expectations_store_name):
+            self._suites = SuiteFactory(
+                store=self.expectations_store,
+                include_rendered_content=self._determine_if_expectation_suite_include_rendered_content(),
+            )
+        else:
+            self._suites = None
```

Comment:
Is this not an error state?

Reply:
Unfortunately there's a set of tests which use a context without an `ExpectationsStore` - I'm not sure that's intended behavior, but would suggest scoping to another ticket if we decide to change it.

Reply:
thanks!